requirements.txt

```diff
@@ -75,6 +75,7 @@ typing-extensions==3.7.4.3
     #   yarl
 yarl==1.6.3
     # via aiohttp
+youtube_dl==2021.5.16
```

Comment:
No, you shouldn't fix the version in the requirements.txt.

Please put `youtube_dl` in the file requirements.in, and then use pip-compile. See the comment at the top of this requirements.txt to see how it works.

Reply:
Hold on

Reply:
Corrected in https://github.com/UrLab/lechbot/pull/64/commits/d4cd3eb3e7aad7162971e40f2928042716f15b7b